### PR TITLE
Add Google Analytics metrics for auto-renewing loans

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-5909

### CODE REVIEW

**Review by iisa — COMMENTED** [review]
@nsharma123 can you give a few lines on your approach for the analytics piece? the changes touch many files... thanks in advance.

File: src/ia-book-actions.js
Comment (nsharma123) [inline src/ia-book-actions.js:409]: need to cookie change name, Forgot!

File: src/core/services/loan-token-poller.js
Comment (nsharma123) [inline src/core/services/loan-token-poller.js:78]: this.identifer vs `identifier=${this.identifier}???

File: src/core/services/loan-analytics.js
Comment (nsharma123) [inline src/core/services/loan-analytics.js:97]: remvoe this..!

File: src/core/services/loan-analytics.js
Comment (nsharma123) [inline src/core/services/loan-analytics.js:111]: autorenew:browse001-autorenew010 ???

File: src/core/services/loan-analytics.js
Comment (nsharma123) [inline src/core/services/loan-analytics.js:131]: can be removed.!

File: src/core/services/loan-analytics.js
Comment (nsharma123) [inline src/core/services/loan-analytics.js:171]: I don't know.

**Review by iisa — APPROVED** [review]
cr 1:1 in person.
approach looks good. lots of files touched, but made for simple organization. 👍 
looks good to merge after he changes we talked about are in. solid work!

**Comment by iisa** [comment]
neeraj please bump up tag to next patch value after merge, thanks!

File: src/core/services/loan-analytics.js
Comment (iisa) [reply src/core/services/loan-analytics.js:171]: we will keep

**Comment by iisa** [comment]
@nsharma123 please merge in https://github.com/internetarchive/iaux-book-actions/pull/59 into this PR before merging to main line. i had to add some logging and found a place where the toast was being suppressed. thanks in advance

**Pull request merged**
